### PR TITLE
SAK-49267 Site Info: Improve Spanish translations for 'Add Participants' section

### PR DESCRIPTION
**Pull request by V100X**
Jira: [SAK-49267](https://sakaiproject.atlassian.net/browse/SAK-49267)

[SAK-49267]: https://sakaiproject.atlassian.net/browse/SAK-49267?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: site-manage/site-manage-participant-helper/src/bundle/org/sakaiproject/site/tool/participant/bundle/sitesetupgeneric_es.properties [hunk 1]
```diff
@@ -67,9 +67,9 @@ participant.status=Estado de los participantes
 
 #AddConf Vm
 addconf.confirming=Confirmando los participantes a\u00f1adidos
-addconf.finish=Los siguientes ser\u00e1n a\u00f1adidos a su sitio cuando pulse en el bot\u00f3n Finalizar.
-addconf.theywill=Ellos
-addconf.theywillnot=No se enviar\u00e1 un correo de notificaci\u00f3n de la disponibilidad de los mismos en el sitio.
+addconf.finish=Los siguientes ser\u00e1n a\u00f1adidos al sitio cuando pulse en el bot\u00f3n Fin.
+addconf.theywill=Se les enviar\u00e1 un correo informando sobre la disponibilidad del sitio.
```
Comment (mpellicer) [inline site-manage/site-manage-participant-helper/src/bundle/org/sakaiproject/site/tool/participant/bundle/sitesetupgeneric_es.properties:71]: 'Se les enviar\u00e1 a los participantes un......'

File: site-manage/site-manage-participant-helper/src/bundle/org/sakaiproject/site/tool/participant/bundle/sitesetupgeneric_es.properties [hunk 2]
```diff
@@ -67,9 +67,9 @@ participant.status=Estado de los participantes
...
+addconf.theywillnot=No se les enviar\u00e1 un correo informando sobre la disponibilidad del sitio.
```
Comment (mpellicer) [inline site-manage/site-manage-participant-helper/src/bundle/org/sakaiproject/site/tool/participant/bundle/sitesetupgeneric_es.properties:72]: 'No se enviar\u00e1 un correo a los participantes informando sobre,...'

**Review by mpellicer — CHANGES_REQUESTED** [review]
Wording changes

File: site-manage/site-manage-participant-helper/src/bundle/org/sakaiproject/site/tool/participant/bundle/sitesetupgeneric_es.properties [hunk 3]
```diff
@@ -67,9 +67,9 @@ participant.status=Estado de los participantes
 
 #AddConf Vm
 addconf.confirming=Confirmando los participantes a\u00f1adidos
-addconf.finish=Los siguientes ser\u00e1n a\u00f1adidos a su sitio cuando pulse en el bot\u00f3n Finalizar.
-addconf.theywill=Ellos
-addconf.theywillnot=No se enviar\u00e1 un correo de notificaci\u00f3n de la disponibilidad de los mismos en el sitio.
+addconf.finish=Los siguientes ser\u00e1n a\u00f1adidos al sitio cuando pulse en el bot\u00f3n Fin.
```
[the remark this answers is not in the document]
Comment (V100X) [reply site-manage/site-manage-participant-helper/src/bundle/org/sakaiproject/site/tool/participant/bundle/sitesetupgeneric_es.properties:70]: It is implied by the title, but could be added anyway
image

**Comment by V100X** [comment]
Now it's a little more concrete, without being too redundant

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
Gracias Vicent! Thanks a ton @V100X

**Pull request merged**